src/cljs/athens/electron/db_modal.cljs

```diff
@@ -15,6 +15,7 @@
     [athens.util :refer [js-event->val]]
     [athens.views.modal :refer [modal-style]]
     [athens.views.textinput :as textinput]
+    [cljsjs.react.dom]
```

Comment:
This isn't needed after https://github.com/athensresearch/athens/pull/1641 was merged. I think the approach in that PR is the correct one because it doesn't introduce side effects (i.e. this adds a global to window, and it's easy to rely on it by accident on other ns).

Reply:
removed